src/main/java/logic/Parser.java

```diff
@@ -0,0 +1,144 @@
+package logic;
+
+import commands.*;
+import task.Task;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+/**
+ * Deals with making sense of user command
+ */
+
+public class Parser {
+    /**
+     * @param dateTimeStr String of dateTime. E.g. 2/12/2019 1800
+     * @return LocalDateTime variable that is stored in Deadline/Event Obj
+     * @throws DukeException If given dateTimeStr is of an incorrect format
+     */
+    public static LocalDateTime parseDateTime(String dateTimeStr) throws DukeException {
+        try {
+            String[] strSplit = dateTimeStr.split(" ");
+            String[] date = strSplit[0].split("/");
+            String time = strSplit[1];
+
+            int year = Integer.parseInt(date[2]);
+            int month = Integer.parseInt(date[1]);
+            int day = Integer.parseInt(date[0]);
+            int hours = Integer.parseInt(time.substring(0, 2));
+            int mins = Integer.parseInt(time.substring(2));
+
+            return LocalDateTime.of(year, month, day, hours, mins);
```

Comment:
May consider using a built-in formatter
https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html